satzung.md

```diff
@@ -17,6 +17,7 @@ Satzung des noch zu benennenden de-RSE Vereins
     1. Der Satzungszweck wird verwirklicht insbesondere durch
         - Durchführung wissenschaftlicher Veranstaltungen zum Themenkomplex Software in der Wissenschaft
         - Erstellung und Publikation (auch elektronisch) von Informations- und Schulungsmaterial zum Themenkomplex Software in der Wissenschaft
+        - Finanzielle Förderung von Forschungsvorhaben zum Thema Software in der Wissenschaft
```

Comment:
Wenn ich recht erinnere, dann ohne "Finanzielle"

Reply:
Ja, hier geht es aber auch nur um die Form. Die konkreten Vorschläge sind in #10.